packages/flutter/lib/src/widgets/container.dart

```diff
@@ -294,17 +294,16 @@ class Container extends StatelessWidget {
   ///
   /// The `height` and `width` values include the padding.
   ///
-  /// The `color` argument is a shorthand for `decoration: new
-  /// BoxDecoration(color: color)`, which means you cannot supply both a `color`
-  /// and a `decoration` argument. If you want to have both a `color` and a
-  /// `decoration`, you can pass the color as the `color` argument to the
-  /// `BoxDecoration`.
+  /// The `color` and `decoration` arguments cannot both be supplied, since
+  /// it would potentially result in the decoration drawing over the background
+  /// color. To supply a decoration with a color, use `decoration: new
```

Comment:
drop the new here since it is no optional?

Reply:
Done